.rubocop.yml

```diff
@@ -39,9 +39,13 @@ Style/FrozenStringLiteralComment:
 Style/NamedSubject:
   Enabled: false
 
+Style/SymbolArray:
+  Exclude: 
+    - 'db/migrate/20210630144922_add_index_to_book_author.rb'  
```

Comment:
https://github.com/IhorMoskvyn/Bookshelf/pull/8/files#r662061154

Reply:
delete this block